CHANGES.txt

```diff
@@ -1,3 +1,9 @@
+4.7.1 (Apr 4, 2023)
+- Added SHA for split and segment fetcher in localhost json.
+- Updated `org.yaml.snakeyaml` dependence to 2.0 for fixing a vulnerability.
+- Fixed Redis, changing []dtos.Key to dtos.Key
```

Comment:
Maybe `Redis storage support` or `Redis integration` rather than just `Redis`?

Reply:
Sure, I can change it. Thanks